Makefile

```diff
@@ -1,38 +1,20 @@
-PREFIX=/usr/local
-BINDIR=${PREFIX}/bin
-DESTDIR=
-BLDDIR = build
-BLDFLAGS=
-EXT=
-ifeq (${GOOS},windows)
-    EXT=.exe
-endif
+GOVERSION := go1.19.1
+SHELL := /bin/bash
```

Comment:
Where the shell is used?

Reply:
it's used behind the scenes, instead of default SH